src/latex/agda-latex-macros.sty

```diff
@@ -325,15 +325,48 @@
 
 \newcommand{\just}{\AgdaInductiveConstructor{just}\xspace}
 
-\newcommand{\LedgerAddress}{\AgdaModule{Ledger.Address}\xspace}
-\newcommand{\LedgerCrypto}{\AgdaModule{Ledger.Crypto}\xspace}
-\newcommand{\LedgerEpoch}{\AgdaModule{Ledger.Epoch}\xspace}
-\newcommand{\LedgerGov}{\AgdaModule{Ledger.Gov}\xspace}
-\newcommand{\LedgerGovernanceActions}{\AgdaModule{Ledger.GovernanceActions}\xspace}
-\newcommand{\LedgerPParams}{\AgdaModule{Ledger.PParams}\xspace}
-\newcommand{\LedgerPrelude}{\AgdaModule{Ledger.Prelude}\xspace}
-\newcommand{\LedgerRatify}{\AgdaModule{Ledger.Ratify}\xspace}
-\newcommand{\LedgerTransaction}{\AgdaModule{Ledger.Transaction}\xspace}
+%% TODO: replace the following 30 or so definitions with a single macro,
+%%       something like this:
+%%           \newcommand{\LedgerURL}[1]{%
+%%           \href{https://github.com/IntersectMBO/formal-ledger-specifications/blob/master/src/Ledger/#1.lagda}%
+%%                {\AgdaModule{Ledger.#1}}%
+%%           }
+%%       then we can just use, e.g., \LedgerURL{Chain} instead of requiring a special \LedgerChain macro.
+%%       Unfortunatley, I get a strange LaTeX error when I try the above.
+%%       (Maybe make this an issue later; not urgent enough to do now.)
+\newcommand{\LedgerLedger}{\href{https://github.com/IntersectMBO/formal-ledger-specifications/blob/master/src/Ledger/Ledger.lagda}{\AgdaModule{Ledger.Ledger}}\xspace}
```

Comment:
Making module names links to the module in our repo will make it easier for readers to find the source if they wish.